cjs/interface/event-target.js

```diff
@@ -6,4 +6,22 @@ const EventTarget = (m => /* c8 ignore start */ m.__esModule ? m.default : m /*
 /**
  * @implements globalThis.EventTarget
  */
-exports.EventTarget = EventTarget;
+class DOMEventTarget extends EventTarget {
+  getParent() {
```

Comment:
specs mention this:

> Each EventTarget object also has an associated get the parent algorithm, which takes an event event, and returns an EventTarget object. Unless specified otherwise it returns null.

So here we don't need to do anything with the event and although I've seen code taking this literally by creating `getTheParent` I just chose `getParent`, it could also be `_getTheParent` or something like that.
This return null here and `Node` returns `parentNode` if it makes sense.